.dependabot/config.yml

```diff
@@ -5,4 +5,4 @@ update_configs:
     update_schedule: "daily"
     ignored_updates:
       - match:
-          dependency_name: "tap"
+          dependency_name: "tape"
```

Comment:
Why is it ignored?

Reply:
I have no familiarity with the decision behind ignoring updates to tap. The best I could do is cargo cult this further without destroying it. Happy to remove it as part of this work, if you approve that.

Reply:
I would suspect it is related to tap 14 being Node 8+. Does the same apply to latest tape versions?

Reply:
It does not seem so. The latest tape seems to happily run on Node 6 on the CI. Looks like we can remove the exception. Thanks for doing the investigation!